docker_ssh/cli.py

```diff
@@ -182,6 +187,14 @@ def create_parser(parser):
         help='Environment variable to set the new socket to (default: %(default)s)',
     )
 
+    parser.add_argument(
+        '--ssh-args',
+        dest='ssh_args',
+        action='append',
+        default=None,
```

Comment:
Hey - sorry I didn't mention this. This needs to be `default=[]` as otherwise it will fail to `append`. :)

Also thanks for the contribution - this is useful.